docs/src/content/docs/getting-started/quick-start.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: Quick Start with Synapse SDK
+description: Learn how to install and use the Synapse SDK.
+sidebar:
+  order: 0
+---
+
+The **Synapse SDK** is your gateway to **Filecoin Onchain Cloud** — a decentralized, programmable cloud platform built on Filecoin. This comprehensive guide will walk you through everything you need to know to start building with the SDK.
+
+The Synapse SDK provides an interface to Filecoin's decentralized services ecosystem:
+
+- **🚀 Recommended Usage**: Use the high-level `Synapse` class for a streamlined experience with sensible defaults
+- **🔧 Composable Components**: Import and use individual components for fine-grained control over specific functionality
+
+The SDK handles all the complexity of blockchain interactions, provider selection, and data management, so you can focus on building your application.
+
+### What You'll Learn
+
+By the end of this guide, you'll understand how to:
+- Install and configure the Synapse SDK
+- Connect to Filecoin networks (mainnet and calibration)
+- Upload and download data using decentralized storage
+- Manage payments and allowances
+- Work with different authentication methods
+- Use advanced features like CDN and custom providers
+
+## Installation
+
+The Synapse SDK requires Node.js 18+ and can be installed via npm, yarn, or pnpm:
+
+```bash
+pnpm install @filoz/synapse-sdk ethers
+```
+
+Note: `ethers` v6 is a peer dependency and must be installed separately.
+
+## Quick Start Example
+
+The [`Synapse`](/api/synapse/classes/synapse/) class provides a complete, easy-to-use interface for interacting with Filecoin storage services.
+
+Get started with storage in just a few lines of code:
+
+### 1: Initialize the SDK
+
+First, set up the Synapse SDK with your credentials:
+
+```typescript
+import { Synapse, RPC_URLS } from '@filoz/synapse-sdk'
+
+// Initialize SDK with your private key and RPC endpoint
+const synapse = await Synapse.create({
+  privateKey: '0x...', // Your private key
+  rpcURL: RPC_URLS.calibration.websocket  // Use calibration testnet for testing
+})
+
+console.log('SDK initialized successfully!')
+console.log(`Connected to network: ${synapse.getNetwork()}`)
+```
+
+### 2: Get Test Tokens
+
+Before you can deposit USDFC, you'll need test tokens from faucets. For the calibration testnet:
+
+**Get tFIL tokens:**
+- Visit the [Filecoin Calibration Faucet](https://faucet.calibnet.chainsafe-fil.io/funds.html)
+- Enter your wallet address to receive free tFIL tokens
+- You need FIL for gas fees
+
+**Get test USDFC tokens:**
+- Visit the [Filecoin Calibration USDFC Faucet](https://forest-explorer.chainsafe.dev/faucet/calibnet_usdfc)
+- Enter your wallet address to receive free test USDFC tokens
+- You need USDFC for storage payments
```

Comment:
you can just do https://synapse.filecoin.services/api/filoz/synapse-core/utils/functions/claimtokens/

Reply:
Does the quick start *only* use calibration net?  Could we:

a) Include one for mainnet, that's the primary path we want and have as an option a way to go on testnet (I know that testnet sounds like the place to do development, but I think getting started should be on mainnet, wdyt?  The cost is relatively low and I'd like to get an active dapp on mainnet.  

b) can we incorporate this end point from @hugomrdias ?